.github/workflows/build-and-test-nightly.yml

```diff
@@ -23,6 +23,8 @@ jobs:
       - name: Install Modules
         run: |
           pip install wget
+          pip install tflite-runtime
+          pip install tflite-support
```

Comment:
can we do `pip install wget tflite-runtime ...` instead? see other lines too..

Reply:
done